CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ https://github.com/nwnxee/unified/compare/build8193.20...HEAD
 - Admin: {Get/Set}MaxLevel()
 - Area: NWNX_Area_GetAmbientSound{Day/Night}()
 - Area: NWNX_Area_GetAmbientSound{Day/Night}Volume
+- Area: NWNX_Area_CreateSoundObject()
```

Comment:
Pre-existing, but we don't list `NWNX_Area_` prefix in the changelog. See `Creature:` below. Doesn't matter either way, I'll clean it up when making the release.